docs/troubleshooting/re-register-nodes.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Reregistering a worker node
+weight: 10
+---
+
+
+## Re-register nodes
+
+### Scenario
+
+On Equinix Metal (Packet) if a cluster is updated from `v0.5.0` to `v0.6.0`, it also installs Packet's Cloud Controller Manager (CCM). The nodes added before `v0.6.0` were configured with BGP using Terraform. From `v0.6.0` onwards they are configured to use CCM for BGP setup. During the update, the older nodes won't see BGP resource removal, but it is possible that someone can disable the BGP from the Equinix Metal console. To make the nodes resilient against such human errors, follow the next steps.
```

Comment:
> To make the nodes resilient against such human errors

This is not 100% true, as CCM only acts on CREATE and DELETE at the moment (packethost/packet-ccm#131).